api/src/pcapi/connectors/cgr/cgr.py

```diff
@@ -67,12 +67,15 @@ def reservation_pass_culture(
     return parse_obj_as(cgr_serializers.ReservationPassCultureResponse, response)
 
 
-def annulation_pass_culture(cinema_details: providers_models.CGRCinemaDetails, qr_code: str) -> None:
-    user = settings.CGR_API_USER
-    password = cinema_details.password
+def annulation_pass_culture(
+    cinema_details: providers_models.CGRCinemaDetails, qr_code: str | None = None, token: str | None = None
+) -> None:
+    if not qr_code and not token:
+        raise AssertionError("Either QRCode or token are mandatory")
```

Comment:
```suggestion
        raise AssertionError("Either QRCode or token is mandatory")
```
Sinon on pense que les deux sont nécessaires, alors qu'il y a `Either`, c'est un peu confusant